phpunit.xml

```diff
@@ -19,4 +19,10 @@
             <directory>src/</directory>
         </whitelist>
     </filter>
+    <php>
+        <server name="CLOUDINARY_URL" value="cloudinary://986847964234786:-4lerN8u6EShZLwJjeXLrUS7yAc@dwinzyahj"/>
```

Comment:
@SlimGee , please do not commit your cloud credentials. Now everyone who sees this has access to your cloud.
You can invalidate your credentials in Cloudinary Admin Console.